cloud-native-runtimes/how-to-guides/troubleshooting.hbs.md

```diff
@@ -16,26 +16,93 @@ API server says: admission webhook "validation.webhook.serving.knative.dev" deni
 
 ### Explanation
 
-Knative adds annotations to Knative Services. kapp controller, which is the orchestrator underneath
-workloads, validates that created objects are not modified by others.
-This conflict between kapp controller and Knative is a known issue and expected behavior that is mitigated
-by a kapp config added during deploy time.
-The kapp config specifies that the annotations Knative adds are correct and can pass validation.
-This configuration is added dynamically in the delivery supply chain.
-
-As of Tanzu Application Platform v1.6.4, the kapp config moved from the delivery supply chain to the
-build supply chain.
-When a web workload is being updated, the delivery supply chain is missing the kapp config, which causes
-the validation error. Though the kapp config exists on v1.6.4 in a different part of the supply chain,
-existing deliverables are not rebuilt to include it.
+Kapp controller, which is the orchestrator underneath workloads, deploys resources exactly as requested.
+Knative adds annotations to Knative Services to track the creator and last modified time of a resource. This
+conflict between kapp controller and Knative is a known issue and expected behavior that is mitigated by a 
+kapp configuration defined and used during deploy time by the delivery supply chain. The kapp config specifies
+that the annotations Knative adds are not to be modified during updates.
+
+However, as of Tanzu Application Platform v1.6.4, the kapp configuration moved from the delivery supply chain to the
+build supply chain. When a web workload is being updated, the delivery supply chain no longer provides the
+kapp configuration, which causes the validation error. Though the kapp configuration exists on v1.6.4 in
+a different part of the supply chain, existing deliverables are not rebuilt to include it.
 
 ### Solution
 
-To workaround this issue, delete and recreate the deliverable. After you recreate the web workload,
```

Comment:
Is it worth providing this as a faster alternative solution if downtime is acceptable to them?

Reply:
Applying the workaround should be quick and shouldn't disrupt existing/running services.